code/modules/clothing/suits/armor.dm

```diff
@@ -442,6 +442,16 @@
 		disable(rand(2, 5))
 		return TRUE
 
+/obj/item/clothing/suit/armor/reactive/random //Spawner for random reactive armor
+	name = "Random Reactive Armor"
+
+/obj/item/clothing/suit/armor/reactive/random/New()
+	..()
+	var/list/types = subtypesof(/obj/item/clothing/suit/armor/reactive)
+	var/A = pick(types)
+	new A(loc)
```

Comment:
you can do this a little cleaner without the `types` var. instead just do `var/spawn_path = pick(subtypesof(...))`